MOcov/mocov_find_files.m

```diff
@@ -67,7 +86,9 @@
         if ~ignore_fn
             path_fn=fullfile(root_dir, fn);
 
-            if isdir(path_fn)
+            if ~isempty(regexp(fn,exclude_re,'once'));
+                continue;
+            elseif isdir(path_fn)
```

Comment:
Line 92 seems to need also the 4th argument (exclude_re) to the ``find_files_recursively`` call.